api/src/database_client.js

```diff
@@ -1,6 +1,9 @@
+import dotenv from "dotenv";
```

Comment:
No need for this `import` as it is already imported in `index.js`, on the top of the file.